### PR TITLE
Added description for each main section

### PR DESCRIPTION
**Pull request by ivo-ritense**
Added a paragraph to each main section about what that section contains and what the target audience is.

### CODE REVIEW

File: extending-valtimo/extending-valtimo.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Extending Valtimo
 
+This section contains information on how to use Valtimo features for which additional code is required. These are 
+usually features that have been set up in a way so that the default functionality can be easily extended. This section 
+is intended for people who want to get more out of Valitmo and are able to write Java or Kotlin code.
```
Comment (marijnritense) [inline extending-valtimo/extending-valtimo.md:5]: Valitmo -> Valtimo.

Maybe mention here all examples will be given in Java?

File: extending-valtimo/extending-valtimo.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Extending Valtimo
 
+This section contains information on how to use Valtimo features for which additional code is required. These are
```
Comment (marijnritense) [inline extending-valtimo/extending-valtimo.md:3]: Maybe rephrase this like:

This section contains instructions and examples on how to extend functionality found in Valtimo, generally through the use of code.

File: extending-valtimo/extending-valtimo.md (hunk 1 above)
Comment (marijnritense) [reply extending-valtimo/extending-valtimo.md:5]: And maybe give an example on what kind of stuff to expect in this section (e.g. creating custom form field data resolvers).

File: reference/reference.md [hunk 3]
```diff
@@ -1,5 +1,12 @@
 # References
 
+This section gives a more complete overview of several things that are available in Valtimo like:
```
Comment (marijnritense) [inline reference/reference.md:3]: Things feels really vague here, but I'm having a hard time coming up with an alternative...

File: valtimo-implementation/valtimo-implementation.md [hunk 4]
```diff
@@ -1,5 +1,9 @@
 # Create your own Valtimo implementation
 
+This section contains information on how to create a new Valtimo implementation. It contains information on which 
+modules are available and what settings at minimum are required to use them. This section is meant for people who want 
+to set up their own Valtimo installation. Some knowledge on java build tools like maven or gradle is aassumed.
```
Comment (marijnritense) [inline valtimo-implementation/valtimo-implementation.md:5]: aassumed -> assumed.

Can you also include links to maven and gradle documentation?

File: using-valtimo/using-valtimo.md [hunk 5]
```diff
@@ -1,5 +1,9 @@
 # Using Valtimo
 
+This section contains information on how to configure and use Valtimo. Any feature included in this section should be 
+usable with no or minimal code required. This section is intended for system administrators that needs to set up
```
Comment (marijnritense) [inline using-valtimo/using-valtimo.md:4]: that needs -> who need

File: extending-valtimo/extending-valtimo.md (hunk 1 above)
Comment (ivo-ritense) [reply extending-valtimo/extending-valtimo.md:5]: > Maybe mention here all examples will be given in Java?

Examples currently are in java or kotlin. These languages are already mentioned. How would you want to do this?

File: reference/reference.md (hunk 3 above)
Comment (ivo-ritense) [reply reference/reference.md:3]: I wasn't very happy with this either, but i was struggling to think of something good.

Comment (theo-ritense) [reply reference/reference.md:3]: 'This section provides an index to features and functionality within Valtimo'?

**Review by Rick-Ritense — APPROVED** [review]
